app/assets/stylesheets/panel/panel.scss

```diff
@@ -35,4 +35,28 @@
       }
     }
   }
-}
\ No newline at end of file
+
+  &.create {
+    .panel-top {
```

Comment:
Since .panel-top is on the same line as .create, you should make it &.panel-top.